src/containers/app/AppActionFactories.js

```diff
@@ -0,0 +1,47 @@
+/*
+ * @flow
```

Comment:
If you're not using Flow annotations, don't include the `@flow` comment block so that Flow can skip this file.

Reply:
+1